.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: python
+dist: xenial
 python:
+- '3.7'
 - '3.6'
 - '3.5'
 - '3.4'
 sudo: required
-services:
-- rabbitmq
+addons:
+  apt:
+    packages:
+      - rabbitmq-server
```

Comment:
👍 for this.  Much better approach overall.

Reply:
and was definitely not done because xenial didn't play nice with the old way of doing it... ^^